unit_tests/player/test_env_player.py

```diff
@@ -270,8 +270,6 @@ def test_get_opponent():
 def test_set_opponent():
     player = CustomEnvPlayer(None, start_listening=False)
     assert player._opponent is None
-    with pytest.raises(RuntimeError):
```

Comment:
Why remove this?

Reply:
The `set_opponent` method used to raise a `RuntimeError` if its argument was not either of the `Player` or `str` type, but since we have type annotations for the arguments of all methods (including `set_opponent`), we no longer need such a test. Otherwise, we might as well be doing such a test for every argument of every method, but that would be overdoing it.